src/components/00-materials/README.md

```diff
@@ -11,10 +11,10 @@ npm install @axa-ch/materials
 ```
 
 ```js
-import { svg } from 'lit-html';
+import { unsafeHTML } from 'lit/directives/unsafe-html';
 import { Arrow_forwardSvg } from '@axa-ch/materials/icons/material-design';
 
-<span>${svg(Arrow_forwardSvg)}</span>;
+<span>${unsafeHTML(Arrow_forwardSvg)}</span>;
```

Comment:
Apparently 'unsafeSVG' does also exist. Would this be an alternative?

Reply:
no, because it is provided for SVG **parts**, but we're passing an entire SVG with top-level `<svg>`!